app/views/shared/_footer_lite.html.slim

```diff
@@ -1,21 +1,49 @@
 footer.footer.bg-light-blue.sm-bg-navy
-  .container.h6.py1.px2.lg-px0
-    .clearfix
-      .col-right.caps
+  .sm-hide.border-bottom
+    .container.py1.px2.lg-px0.h5
+      .center
+        span.i18n-mobile-toggle.block.text-decoration-none.blue.fs-13p
+          = image_tag asset_url('globe-blue.svg'), width: 12, class: 'mr1', alt: ''
+          = t('i18n.language')
+          span.caret.inline-block.ml-tiny
+            | &#9662;
+  .i18n-mobile-dropdown.sm-hide
+    ul.list-reset.mb0.white.center
+      li.border-bottom
+        = link_to t('i18n.locale.en'), { locale: 'en' },
+          class: 'block py-12p px2 text-decoration-none blue fs-13p'
+      li.border-bottom
+        = link_to t('i18n.locale.es'), { locale: 'es' },
+          class: 'block py-12p px2 text-decoration-none blue fs-13p'
+  .container.py1.sm-py0.px2.lg-px0
+    .flex.flex-center
+      .flex.flex-center
+        .i18n-desktop-toggle.sm-show.white.my1.mr3.px1.py-tiny.border.border-blue.rounded-lg
+          = image_tag asset_url('globe-white.svg'), width: 12, class: 'mr1', alt: ''
+          = t('i18n.language')
+          span.caret.inline-block.ml-tiny
+            | &#9662;
+          .i18n-desktop-dropdown
+            ul.list-reset.mb0.white
+              li.border-bottom.border-navy
+                = link_to t('i18n.locale.en'), { locale: 'en' },
```

Comment:
1. Can we iterate over this with `I18n.available_locales` -- so we get new locales added automatically? (maybe we can add a custom sort somewhere else, but would be good to minimize the number of places we have to update)

2. Can we update these to only show the language selector if there is more than one thing in `I18n.available_locales`? That way we have have the selector show up in lower envs while we QA our languages but then we don't show the selector in prod where we have other languages disabled temporarily